scripts/kubestellar

```diff
@@ -320,7 +320,7 @@ if [[ ! -d $log_folder ]]; then
     mkdir -p $log_folder
 fi
 
-mailbox-controller -v=2 >& $log_folder/mailbox-controller-log.txt &
+mailbox-controller -v=2 >>& $log_folder/mailbox-controller-log.txt &
```

Comment:
Are you sure about these?  https://www.gnu.org/software/bash/manual/bash.html#Appending-Standard-Output-and-Standard-Error says it should be `&>>`.  And that is what I get when testing:

```console
mspreitz@mjs12 ~ % bash
bash-5.2$ ls -ld fubar
ls: fubar: No such file or directory
bash-5.2$ date >>& fubar
bash: syntax error near unexpected token `&'
bash-5.2$ date &>> fubar
bash-5.2$ ls -ld fubar
-rw-r--r--  1 mspreitz  staff  29 Aug 16 17:44 fubar
```

Reply:
changed all 3 to match your example.